src/types.ts

```diff
@@ -165,12 +165,34 @@ export interface GetHomeworkOptions {
 	 */
 	to?: string;
 }
+/**
+ * Teacher uploaded attachments
+ */
 export interface ValidatedHomeworkAttachment {
 	id: number;
 	file_name: string;
 	file: string;
 	validated_file: string;
 }
+/**
+ * Teacher uploaded links
+ */
+export interface ValidatedLink {
+  link: string;
+  validated_link: string;
+}
+/**
+ * User uploaded attachments
+ */
+export interface HomeworkAttachment {
```

Comment:
```suggestion
export interface StudentHomeworkAttachment {
```

Preferably rename this to something where it's clear that it's a student attachment, without needing to look at the comment. Doesn't need to be `StudentHomeworkAttachment` just as long as it's clear, and happy to hear other opinions on this.